spec/models/user_mentee_application_spec.rb

```diff
@@ -105,9 +105,9 @@
     it 'enqueues application submission notifications after create commit' do
       expect {
         create(:user_mentee_application)
-      }.to have_enqueued_mail(UserMenteeApplicationMailer, :notify_for_application_submission)
-        .and have_enqueued_mail(UserMenteeApplicationAlertMailer,
```

Comment:
I removed `UserMenteeApplicationAlertMailer` altogether, as it didn't really convey how/why the mailer would get invoked.